android/FlurryPlugin.java

```diff
@@ -27,24 +29,37 @@
 
 import com.flurry.android.FlurryAgent;
 
+import static android.util.Log.VERBOSE;
+
 
 public class FlurryPlugin implements IPlugin {
     Activity activity;
     String flurryAppKey = "";
     boolean debug = false;
+    String TAG = "FlurryPlugin";
 
     public FlurryPlugin() {
 
     }
 
-    private void initFlurry(Context context) {
+    private void initFlurry(Context context, String flurryAppKey) {
+        FlurryAgent.Builder fab = new FlurryAgent.Builder()
+                .withLogEnabled(true)
+                .withListener(new FlurryAgentListener() {
+                    @Override
+                    public void onSessionStarted() {
+                        Log.d(TAG,"FlurryAgent session started.");
+                    }
+                })
+                .withCaptureUncaughtExceptions(true)
+                .withContinueSessionMillis(10000)
+                .withLogLevel(VERBOSE);
```

Comment:
will this always print a lot of log?
It might be useful to just do it when debug is set to true

Reply:
Log.d (Log.debug) is removed if release is assembled

Reply:
okay great

Reply:
Ok

Reply:
This will not print log always but I moved `.withLogLevel(VERBOSE)` to debug section.